src/main/java/net/fabricmc/loom/extension/LoomGradleExtensionApiImpl.java

```diff
@@ -238,6 +238,10 @@ public void addMinecraftJarProcessor(Class<? extends MinecraftJarProcessor<?>> c
 
 	@Override
 	public Dependency officialMojangMappings() {
+		if (notObfuscated()) {
+			throw new UnsupportedOperationException("Cannot use Mojang mappings in a none obfuscated environment");
```

Comment:
```suggestion
			throw new UnsupportedOperationException("Cannot use Mojang mappings in a non-obfuscated environment");
```
Or maybe "unobfuscated"